packages/design-system/src/components/web-components/preactement/define.ts

```diff
@@ -107,8 +109,8 @@ function createCustomElement<T>(
       onDisconnected.call(this);
     }
 
-    public renderPreactComponent(addedNodes?: Node[]) {
-      renderPreactComponent.call(this, addedNodes);
+    public forceRender(addedNodes?: Node[]) {
```

Comment:
Forgot to call out that I changed this name. What do you all think about this? It's more generic, and since we're exposing it, maybe it should clearly state what it does from an external perspective (forces a re-render)?